src/ray/raylet/node_manager.cc

```diff
@@ -1081,12 +1081,15 @@ void NodeManager::HandleWorkerAvailable(
 void NodeManager::HandleWorkerAvailable(const std::shared_ptr<Worker> &worker) {
   RAY_CHECK(worker);
   // If the worker was assigned a task, mark it as finished.
+  bool worker_available = true;
```

Comment:
Consider bool is_actor

Reply:
Hmm yeah I'll call it is_direct_actor since we don't actually do that for raylet actors.

Reply:
Or I guess is_not_direct_actor...